styles/helpers/_visibility.scss

```diff
@@ -0,0 +1,4 @@
+.hide-off-page {
+  position: absolute;
+  left: -999em;
```

Comment:
Is it polite to add display: none here, so screen readers don't keep trying to read it out - or will that prevent events from happening - not sure where this will be used.

Reply:
Good shout on screen readers. However we'll resolve that by making them invisible via `aria-hidden=true`. Those roles will be applied when writing them elements themselves...nothing in this PR.

`display:none` causes multiple issues like not being able to access the node in the DOM. Hence the name of the class...we're not just hiding the element, we're sending it off page.